sale_commission/__openerp__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'Sales commissions',
-    'version': '8.0.2.4.0',
+    'version': '8.0.2.4.1',
```

Comment:
It should be 8.0.3.0.0 as it includes a major change with migration script (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)